script/csv_import/metric/metric_csv_row.rb

```diff
@@ -37,10 +37,8 @@ def normalize_value_type value
     value.match(/(?<type>[^(]+)\((?<options>[^)]+)/) do |match|
       new_value = match[:type].strip
       new_value = "Category" if new_value == "Categorical"
-      if new_value == "Category" || new_value == "Multi-Category"
+      if %w[Category Multi-Category].include? new_value
```

Comment:
on my hit list this is the least preferable option. Maybe we can agree on `new_value.in? %w[Category Multi-Category]`?